docker-compose.yml

```
@@ -5,13 +5,13 @@ services:
     image: postgres:12-alpine
     ports:
       - 5432:5432
-    volumes:
-      - .data/postgres:/var/lib/postgresql/data
+    # volumes:
+    #   - .data/postgres:/var/lib/postgresql/data
     environment:
       POSTGRES_PASSWORD: postgres
 
   subquery-node:
-    image: onfinality/subql-node:latest
+    image: docker.soramitsu.co.jp/sora2/subquery:dev
```

```suggestion
    image: local/subquery:dev
    build: .
```
the repository is public but the image is private,